lib/email/reader.rb

```diff
@@ -0,0 +1,51 @@
+class Email::Reader
```

Comment:
Use nested module/class definitions instead of compact style.<br>Missing frozen string literal comment.